managed/src/main/java/com/yugabyte/yw/scheduler/Scheduler.java

```diff
@@ -251,6 +254,17 @@ private void runMultiTableBackupsTask(Schedule schedule) {
   }
 
   private void runDeleteBackupTask(Customer customer, Backup backup) {
+    UUID universeUUID = backup.getBackupInfo().universeUUID;
+    try {
+      Universe universe = Universe.get(universeUUID);
+      if (universe.getUniverseDetails().universePaused) {
+        LOG.warn("Cannot delete the backup since the universe {} is currently in a paused state.",
+            universeUUID.toString());
+        return;
+      }
+    } catch (Exception e) {
+      LOG.warn("Universe not found for the backup {}.", backup.backupUUID.toString());
```

Comment:
I believe backup deletion does not need the universe. LMK if I have to return here.

Reply:
Above suggestion makes it a moot question. Because getExpired will check if there is universe && it is paused skip that. But if there is no universe it will return that backup. 
Most likely that deletion will fail but we are good on that due to the other fix I made earlier this week.

Reply:
yeah, the check is present. Adding paused check there only.